apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch/index.tsx

```diff
@@ -57,11 +61,18 @@ const Launch: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 
 	const CurrentLaunchStep = LaunchStepComponents[ currentStep ];
 
+	React.useEffect( () => {
+		if ( title === __( 'Site Title', 'full-site-editing' ) ) {
+			// Clear title value if the site was created with the fallback title. This will also set the first step as incomplete.
+			saveTitle( '' );
```

Comment:
I expect this not clear it from the actual site? I.e. the site title should be modified by customer, even if it's default one or empty.

Reply:
Yeah, you're right. As long as we allow editing it we should leave it.